sig-cli/charter.md

```diff
@@ -11,7 +11,86 @@ libraries to interface with Kubernetes API's.
 
 ### In scope
 
-SIG CLI [README]
+#### Transparently Exposing APIs through Declarative and Imperative Workflows
+
+The scope of CLI Tools focuses on enabling declarative and imperative workflows
+for invoking kubernetes APIs and authoring Resource Config.  Tools provide
+commands for both generalized (e.g. create resource from Resource Config) tasks and
+specialized (e.g. drain a node, exec into a container) tasks.
+
+It is the philosophy of the tools developed in SIG CLI to facilitate working
+directly with the Kubernetes APIs and Kubernetes style Resources, and to the
+extend possible, provide a transparent experience for how commands map to
+Kubernetes APIs and Resources.
+
+Building tools that obfuscate the underlying Resources and APIs (e.g. through
+generalized templating or DSLs) is an anti-goal of SIG CLI.  Notable examples
+of commands that violate this principle: `run`, `expose`, `autoscale`
```

Comment:
I'm not sure what having this in the charter scope means.  Does that mean it is an explicit and fundamental goal of sig-cli to _remove_ these subcommands from `kubectl`?  (I'm ok with that, it just seems odd to call that out as a charter scope)

If it _is_ an explicit goal to remove anything that "abstracts the underlying resources and APIs", then we should probably carefully consider the wording of this particular phrase.  Eg: `describe`, `rollout`, `kustomize` are some other subcommands that don't map directly onto the REST API, but it might be surprising to have them suddenly in-scope for removal.

(tl;dr: I think the direction that this paragraph is trying to convey is unclear)

Reply:
I was trying to convey that building out new abstractions is not a goal.

Reply:
I just deleted that section in my pending update.